llvm/tools/llvm-cas/llvm-cas.cpp

```diff
@@ -245,6 +250,29 @@ int main(int Argc, char **Argv) {
   return catBlob(*CAS, ID);
 }
 
+int ingestCasIDFile(cas::ObjectStore &CAS) {
+  ExitOnError ExitOnErr;
+  StringMap<ObjectRef> Files;
+  SmallVector<ObjectProxy> SummaryIDs;
+  for (StringRef IF : Inputs) {
```

Comment:
I prefer this `Inputs` to be taken from parameter too.

Reply:
Ah sorry, I missed that one!